server/db/repository/WishCardRepository.js

```diff
@@ -23,6 +23,8 @@ async function getViewableWishCards(showDonated) {
   try {
     const searchArray = [];
     searchArray.push({ status: 'published' });
+    // Patric told me to add this line -Stacy
+    searchArray.push({ status: 'draft' });
```

Comment:
```js
const searchArray = [{ status: 'published' }, { status: 'draft' }];
```
and remove the other 3 lines

Reply:
changed it

Reply:
@Enubia can you merge this once the checks pass?